Microsoft.Toolkit.Uwp.UI.Controls.Layout/BladeView/BladeView.xaml

```diff
@@ -89,7 +89,7 @@
         <Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />
         <Setter Property="FontSize" Value="{ThemeResource ControlContentThemeFontSize}" />
         <Setter Property="TitleBarBackground" Value="{ThemeResource SystemControlBackgroundChromeMediumLowBrush}" />
-        <Setter Property="Background" Value="{ThemeResource SystemControlBackgroundAltHighBrush}" />
+        <Setter Property="Background" Value="{ThemeResource CardBackgroundFillColorDefaultBrush}" />
         <Setter Property="Foreground" Value="{ThemeResource SystemControlForegroundBaseHighBrush}" />
```

Comment:
It seems to look a lot better, on dark and light system themes.
Found it here 
https://github.com/microsoft/microsoft-ui-xaml/blob/5e815a2c4bfe2bc019562b01fffbeeb25866acf6/dev/CommonStyles/Common_themeresources_any.xaml#L185

Reply:
Looks good in Dark theme, Light theme ... a little less so.  No telling if/when WinUI will move the goal posts on Light theme.